### DISCUSSION TITLE
Maintain the template using two UI Libraries together or in separated repos or...?

### DISCUSSION THREAD

**Discussion by yukinoda** [monstar-lab-oss/admin-panel-template-reactjs#62] [General] [answered]
We currently have two UI Libraries in this repo
- Material-UI
- Ant-Design

**Comment by geekonthemove** [comment, marked as answer]
I vote for deleting the Material UI branch 👍

**Reply by yukinoda** [reply]
Same, the `material-ui` branch is already "too" deprecated. It would be better to start a new repo from scratch if we are to up to create a `material-ui` admin panel template
